form.js

```diff
@@ -0,0 +1,32 @@
+// const form = document.querySelector('.form');
+// const name = document.querySelector('#name');
+// const email = document.querySelector('#email');
+// const message = document.querySelector('#message');
+// const submit = document.querySelector('#submit');
+
+// form.addEventListener('submit', (e) => {
+//   // prevent the form from submitting
+//   e.preventDefault();
+// });
+
+// const isEmailValid = (email) => {
+//   const re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\
+// [\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]
+// {1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+//   return re.test(email);
```

Comment:
- [ ] please try to remove the unused commented code in order to keep the file clean and pro